aasemble/django/apps/buildsvc/templates/buildsvc/html/builds.html

```diff
@@ -22,6 +24,14 @@
           <td><a href="#">{{ build.source.name }}</a></td>
           <td><a href="#">{{ build.version }}</a></td>
           <td><a href="#">{{ build.build_started|naturaltime }}</a></td>
+          <td><a href="#">{{ build.build_finished|naturaltime }}</a></td>
+          <td><a href="#">
+              {% if build.build_finished %}
+                  Finished
+              {% else %}
+                  Running
+              {% endif %}
+          </a></td>
```

Comment:
Builds from before we started tracking build_finished will show as "Running". Are you ok with that, or do you have thoughts on how to avoid it?